### PR TITLE
Improve vcf info tests

### PR DESCRIPTION
**Pull request by xbrianh**
depends on #194

### CODE REVIEW

File: terra_notebook_utils/vcf.py [hunk 1]
```diff
@@ -88,6 +87,18 @@ def with_file(cls, filepath, read_buf: memoryview=None):
...
+    if uri.startswith("drs://"):
+        client, drs_info = drs.resolve_drs_for_gs_storage(uri)
+        blob = client.bucket(drs_info.bucket_name, user_project=google_billing_project).get_blob(drs_info.key)
+        return VCFInfo.with_blob(blob)
+    if uri.startswith("gs://"):
+        bucket, key = uri[5:].split("/", 1)
+        blob = gs.get_client().bucket(bucket, user_project=google_billing_project).get_blob(key)
+        return VCFInfo.with_blob(blob)
+    else:
+        return VCFInfo.with_file(uri)
```
Comment (DailyDreaming) [inline terra_notebook_utils/vcf.py:101]: Should this check (and potentially fail with `NotImplementedError`) for non-local schema like `s3://`?

File: tests/infra/__init__.py [hunk 2]
```diff
@@ -15,7 +16,15 @@ def setUp(self):
...
+    sys.stdout = io.TextIOWrapper(io.BytesIO(), sys.stdout.encoding)
     yield
     sys.stdout.close()
     sys.stdout = old_stdout
+
+def upload_data(uri: str, data: bytes):
+    if uri.startswith("gs://"):
+        bucket, key = uri[5:].split("/", 1)
+        Client().bucket(bucket).blob(key).upload_from_file(io.BytesIO(data))
+    else:
```
Comment (DailyDreaming) [inline tests/infra/__init__.py:28]: Same as above, though possibly checking for `drs://` if this is never expected to run `drs` (and maybe a comment about why no drs here).

**Review by DailyDreaming — APPROVED** [review]
LGTM.  See comments.

File: terra_notebook_utils/vcf.py (hunk 1 above)
Comment (xbrianh) [reply terra_notebook_utils/vcf.py:101]: Will add

File: tests/infra/__init__.py (hunk 2 above)
Comment (xbrianh) [reply tests/infra/__init__.py:28]: This is not a user space function, but I'll add a "not supprted".

Devs attempting to upload to drs uris have deep misconceptions.

Comment (xbrianh) [reply tests/infra/__init__.py:28]: On second thought, I'm going to leave this as-is. Adding arbitrary checks for hypothetical developer misconceptions doesn't seem worth the complexity.

**Pull request merged**